exchange/utils.go

```diff
@@ -90,9 +91,10 @@ func (rs *requestSplitter) cleanOpenRTBRequests(ctx context.Context,
 
 	var gpp gpplib.GppContainer
 	if req.BidRequest.Regs != nil && len(req.BidRequest.Regs.GPP) > 0 {
-		gpp, err = gpplib.Parse(req.BidRequest.Regs.GPP)
-		if err != nil {
-			errs = append(errs, err)
+		var gppErrs []error
+		gpp, gppErrs = gpplib.Parse(req.BidRequest.Regs.GPP)
+		if len(gppErrs) > 0 {
+			errs = append(errs, gppErrs[0])
```

Comment:
Same here. Given that lines 67 and 82 above append return all the errors found in `parseAliases(req.BidRequest)` and `parseAliasesGVLIDs(req.BidRequest)` respectively, I'd favor to simply append all of the errors `gpplib.Parse(req.BidRequest.Regs.GPP)` throws. Toughts?